addons/mail/static/src/models/notification_group_manager/notification_group_manager.js

```diff
@@ -48,8 +48,10 @@ function factory(dependencies) {
                 } else {
                     date = moment.max(group.date, notification.message.date);
```

Comment:
I just noticed, but also this is using message.date as if it was always set, which might not be the case now.
We probably need to also handle a group having no date too then.

Reply:
Group date are already guarded inside the template: https://github.com/odoo-dev/odoo/blob/master-mail-remove-message-default-date-did/addons/mail/static/src/components/notification_group/notification_group.xml#L21

Reply:
I maked the group date a compute, it look better like this